src/EasyList/Program.cs

```diff
@@ -5,7 +5,7 @@ namespace EasyList
 {
     class Program
     {
-        public static ITodoService _todoService = Factory.CreateTodoServiceDB();
+        public static ITodoService TodoService => Factory.CreateTodoServiceDB();
```

Comment:
Correct naming convention.

Reply:
This is a field and not a Property so _todoService ? @Anequit

Reply:
It is actually a property. This is shorthand for a read-only property.

Reply:
Why is the service a static instance and inside program.cs if its not being used here?

Reply:
> 
> 
> Why is the service a static instance and inside program.cs if its not being used here?

So that is becomes very easy to identify where we declared it at one place.

Reply:
> 
> 
> It is actually a property. This is shorthand for a read-only property.

@valincius I meant before changing it. As you suggested last time i changed it to a read only property [here](https://github.com/Code2Gether-Discord/EasyList/pull/25#discussion_r688742898)

Reply:
> > Why is the service a static instance and inside program.cs if its not being used here?
> 
> So that is becomes very easy to identify where we declared it at one place.

What do you mean with "identify"? I have mentioned this before. You have set up a dependency inversion system with the factories, i would stick with that. You should first of all use encapsulation which is part of the OOP principles. Secondly there is no point in making a class non-static just to make it static afterwards. Also always try to keep Program.cs as clean as possible, it is only supposed to be used as the starting point of your application.

Reply:
Makes sense